OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -211,8 +211,26 @@ public void OnAllRemovedFromWorld(Actor[] actors, LuaFunction func)
 						return;
 
 					if (!group.Any())
-						using (f)
-							f.Call().Dispose();
+					{
+						// Do not dispose the 'original' function f, we may need it again
+						var temp = (LuaFunction)f.CopyReference();
+						using (temp)
+							temp.Call().Dispose();
+					}
+				}
+				catch (Exception e)
+				{
+					Context.FatalError(e.Message);
+				}
+			};
+			Action<Actor> onMemberAdded = m =>
```

Comment:
Can you explain the purpose of this?

Reply:
Also style nit: empty line after `};`

Reply:
We want to add actors back into the "yet to be removed from world" group in case they get added to the world again. Take loading units in and out of a transport, for example.

Reply:
Resolved the style nit.